charts/agent-control/values.yaml

```diff
@@ -34,84 +34,46 @@ agent-control-deployment:
-  # named `HelmRelease` that the operator will reconcile on the apply, edit, or deletion of a `HelmRelease` resource.
-  #
-  # New Relic' Agent Control will use this controller by creating `HelmRelease` CRs based in the configuration stored on
-  # Fleet Control. This is the only controller that the Agent Control need for now, the other controllers are disabled by default.
-  #
-  # On the other hand, user might want to leverage having FluxCD installed for their own purposes. Take a look to the
-  # `values.yaml` to see how to enable other controllers.
-  # @default -- Enabled
-  helmController:
-    create: true
-    annotations:
-      prometheus.io/port: "8080"
-      prometheus.io/scrape: "false"
-
-  # This controller is exactly the same as the Helm controller (refer to it) but assembling manifests with
-  # Kustomize instead of using templating systems like Helm.
-  kustomizeController:
-    create: false
-
-  # The image automation controller updates YAML files based on the latest images scanned by image reflector
-  # controller, and commits the changes to a given Git repository.
-  imageAutomationController:
-    create: false
-
-  # The image reflector controller scans image repositories and reflects the image metadata in Kubernetes resources
-  # ready to be used by other controllers.
-  imageReflectionController:
-    create: false
-
-  # The notification controller handles events coming from external systems (GitHub, GitLab, Bitbucket, Harbor,
-  # Jenkins, etc) and notifies the GitOps toolkit controllers about source changes. The controller also handles events
-  # emitted by the GitOps toolkit controllers (source, kustomize, helm) and dispatches them to external systems (Slack,
-  # Microsoft Teams, Discord) based on event severity and involved objects.
-  #
-  # New Relic provides a powerful alert system with multiple policies and routes to alert users so it is disabled by
-  # default on our FluxCD distribution.
-  notificationController:
-    create: false
-
-  # Upstream chart create Network Policies. They are relaxed to enough to not cut any malicious attack and not
-  # reduce the attack surface enough on environments where the security is a must.
-  policies:
-    create: false
-
-  # -- Create RBAC rules for FluxCD is able to deploy all kind of workloads on the cluster.
-  # @default -- Enabled (See `values.yaml`)
-  rbac:
-    create: true
-    createAggregation: true
-
-  # -- As we are using Flux as a tool from the agent control to release new workloads, we do not want Flux to listen to
-  # all CRs created on the whole cluster. If the user does not want to use Flux and is only using it because of the
-  # agent control, this is the way to go so the cluster has deployed all operators needed by the agent control. But if the
-  # user want to use Flux for other purposes besides the agent control, this toggle can be used to allow Flux to work on
-  # the whole cluster.
-  watchAllNamespaces: false
```

Comment:
Why did we leave these values out? Are we fine with the defaults?

Reply:
Moved to the umbrella-chart!